### PR TITLE
Shemshey maxim dao task

### PR DESCRIPTION
**Pull request by ShemsheyMaxim**

### CODE REVIEW

**Review by MykolaFatkullin — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandlerImpl {
+    private final BetDao betDao = new BetDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java:13]: other Dao?

File: src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            Bet bet;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+
+            } catch (ArrayIndexOutOfBoundsException | NumberFormatException e) {
+                System.out.println("Пожалуйста, введите валидные данные: "
+                        + "сумму(число) и коеффициент(число) через пробел для для вашей ставки");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java]: Let's write in English)

File: src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                continue;
+            }
+            betDao.add(bet);
+            System.out.println(bet);
+            System.out.println("Введите ещё одну ставку или введите \"стоп\" "
+                    + "чтобы завершить работу");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java]: Let's pring all bets using getAll() after loop

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java (hunk 1 above)
Comment (ShemsheyMaxim) [reply src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java:13]: done

File: src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java (hunk 2 above)
Comment (ShemsheyMaxim) [reply src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java]: done

File: src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java (hunk 3 above)
Comment (ShemsheyMaxim) [reply src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Comment by ShemsheyMaxim** [comment]
I forgot merge pull request after first part.

**Review by mfedenko09 — APPROVED** [review]

**Comment by RomanDubovskyi** [comment]
you have files from prev hw, they should not be in this PR, fix it before review

**Pull request merged**